prism.el

```diff
@@ -1041,6 +1056,13 @@ it's extrapolated to the length of `prism-faces'."
   :type '(repeat number)
   :set #'prism-customize-set)
 
+(defcustom prism-opacities '(99 99 99)
```

Comment:
Why is the default value `'(99 99 99)`?

Reply:
Leftover from when I was debugging something, switching to 100.